doc/input-output-reference/src/overview/group-location-climate-weather-file-access.tex

```diff
@@ -567,175 +567,165 @@ \subsubsection{Inputs}\label{inputs-3-020}
+Note: EnergyPlus processed weather files available on the EnergyPlus web site: \url{https://www.energyplus.net/weather}
+have neither special days specified nor daylight saving period. However, DDY (Design Day) files produced
+from the ASHRAE Design Conditions that accompany the EPW files may include a DaylightSavingPeriod object
+for certain locations.
 \end{callout}
 
 \paragraph{Field: Use Weather File Rain Indicators}\label{field-use-weather-file-rain-indicators-1}
 
-Weather files can contain ``rain'' indicators. (EPW field ``Present Weather Codes'' -- described in the AuxiliaryPrograms document). In turn, rain indicates wet surfaces which changes the film convection coefficient for the surface. Other models may use rain as well (Ground Heat Exchangers). Entering ``\textbf{Yes}'' in this field allows the weather file conditions to represent ``Rain''; entering ``\textbf{No}'' in the field ``turns off'' the rain indicator for this period. You might use this to be able to compare two ``same location'' weather files of different years, origins, etc.
+Weather files can contain ``rain'' indicators. (EPW field ``Present Weather Codes'' -- described in the
+AuxiliaryPrograms document). In turn, rain indicates wet surfaces which changes the film convection
+coefficient for the surface. Other models may use rain as well (Ground Heat Exchangers). Entering
+``\textbf{Yes}'' in this field allows the weather file conditions to represent ``Rain''; entering
+``\textbf{No}'' in the field ``turns off'' the rain indicator for this period. You might use this
+to be able to compare two ``same location'' weather files of different years, origins, etc.
 
 \paragraph{Field: Use Weather File Snow Indicators}\label{field-use-weather-file-snow-indicators-1}
 
-Weather files can contain ``snow'' indicators. (EPW field ``Snow Depth'' \textgreater{} 0 indicates ``Snow on the ground''). In turn, snow changes the reflectivity of the ground and cascades changes of this reflectivity. Entering ``\textbf{Yes}'' in this field allows the weather file conditions to represent ``Snow''; entering ``\textbf{No}'' in the field ``turns off'' the snow indicator for this period. You might use this to be able to compare two ``same location'' weather files of different years, origins, etc.
+Weather files can contain ``snow'' indicators. (EPW field ``Snow Depth'' \textgreater{} 0 indicates
+``Snow on the ground''). In turn, snow changes the reflectivity of the ground and cascades changes of this
+reflectivity. Entering ``\textbf{Yes}'' in this field allows the weather file conditions to represent
+``Snow''; entering ``\textbf{No}'' in the field ``turns off'' the snow indicator for this period. You might
+use this to be able to compare two ``same location'' weather files of different years, origins, etc.
+
+\paragraph{Field: Treat Weather as Actual}\label{field-treat-weather-as-actual}
```

Comment:
@jasondegraw A little late in the game to suggest a change here so feel free to ignore this comment. The name of this field confuses me, because it leads me to think more about the weather data values rather than the dates. Even synthesized data could be created with valid date fields that you want to match. For example, one might create an epw with 20 years of projected weather data and want to run specific years from within that epw. Would you be open to something like "Match (or Align) Weather File Years" or "Weather Files Dates Must Match Exactly" or "Strict Weather File Dates"?

Reply:
@mjwitte The current name came about because it triggers the behavior more like what you used to get with RunPeriod:CustomRange that was determined by a field named "ActualWeather" or something like that. Mainly that means that the first entry of start date must appear in the EPW (and that's all) and that if a leap year is covered in the run period the EPW must have a leap day. So I'm not sure about "strict" as a descriptor. How about something like "Use actual calendar dates"? I guess that could lead to other questions, maybe something similar would get across the message.

Reply:
I'm not finding an old field like that, but it's not big deal here. I'm going to take a crack at writing the transition code for RunPeriod:CustomRange.

Reply:
Also, the performance tests need updating.

Reply:
@mjwitte Sorry, it wasn't a field, it is/was a member variable:

https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/WeatherManager.hh#L382